addons/viewdistance/functions/fnc_returnViewDistanceValue.sqf

```diff
@@ -0,0 +1,43 @@
+/*
+ * Author: Winter
+ * Returns the view distance value according to the given index
+ * 
+ *
+ * Arguments:
+ * 0: View Distance Index <SCALAR>
+ *
+ * Return Value:
+ * View Distance <SCALAR>
+ *
+ * Example:
+ * [2] call ace_viewdistance_fnc_returnViewDistanceValue;
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+PARAMS_1(_index);
+
+private ["_return"];
+_return = 0;
+
+// change to if () exitWith {};?
+switch (_index) do
+{
```

Comment:
Our code style uses:
```
switch (_index) do { 
``` 

This is a switch statement, so this will be better:
```
_return = switch (_index) do { 
  case 0: {1500};
  case 1: {2000};
  case 2: {2500};
 ...
  default {1000}; // default value
};
```